### ISSUE TITLE
Feature: Provide a way to override sourceRoot

### ISSUE THREAD

**Issue by penleychan** [ngneat/transloco-keys-manager#148]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Currently the default source root `src` if no workspace/angular.json is found. 

Project was generated with nx with Integrated Repos workspace, and default source root is `apps`.

### Describe the solution you'd like

Provide a configuration to override sourceRoot via options on transloco.config.js.

```
{
   ....
   sourceRoot: 'apps',
   ....
}
```

### Describe alternatives you've considered

```markdown
N/A
```

### Describe alternatives you've considered

N/A

### I would like to make a pull request for this feature

No

**Comment by JelleBruisten** [comment]
This could also provide workaround for the bug #149 since when we call `transloco-keys-manager extract --sourceRoot /libs/my-library` it won't need to find a `angular.json` to figure out the sourceRoot.

**Comment by penleychan** [comment]
No idea when this feature ever going to be implement or if ever, I've quickly created a patch for my own use case no idea if this will work for you or not don't have time to invest in it. Published on npmjs:

https://www.npmjs.com/package/@penleychan/transloco-keys-manager

`yarn add @penleychan/transloco-keys-manager`

Provide `sourceRoot` on `transloco.config.js`:
```
keysManager: {
    ...
    sourceRoot: 'apps'
  },
```
Or via cli
`transloco-keys-manager extract --project PROJECTNAME --sourceRoot apps`

Basically all I did was commented out that error message, it will still default to `src` then on `resolveConfig.ts` If there's `sourceRoot` detected from cli or config.js it will override the `projectBasePath`.

If nothing is found it will display `Input path provided doesn't exists!`

**Comment by shaharkazaz** [comment]
I've added support for workspaces without root configurations, please note that you must pass the project name in order for the commands to work since it can't resolve the default project without a root config.
Pleas try v3.6 and LMK.

**Comment by penleychan** [comment]
Thanks for adding this.

2 Issues:
1. Extract seems to only look within the `apps/[projectname]/src/app` directory, anything outside of that directory does not get scanned ex. `apps/[projectname]/src/components`. But this can be resolved by providing `input` value to `./`
2. Extracted keys are found but it does not populate the json files.

Regards

**Comment by shaharkazaz** [comment]
@penleychan As you said, it's not an issue since it's configurable 🙂 
Closing this as the main issue is resolved.
